index.js

```diff
@@ -1,16 +1,18 @@
 /* eslint-env node */
 'use strict';
 
-const path = require('path');
 const Funnel = require('broccoli-funnel');
 const Merge = require('broccoli-merge-trees');
 const fastbootTransform = require('fastboot-transform');
 const existsSync = require('exists-sync');
 
+const { getFontPaths, getMaterializePaths } = require('./utils.js');
```

Comment:
I moved the code that deals with determining the paths of various materialize and font assets into this new `utils.js`. I don't think we ought to expose this to users though. If we want to give them more control over where the assets are, perhaps they could specify desired paths as options to override our defaults. Thoughts?